doc/user-guide/Getting-started.md

```diff
@@ -55,6 +85,22 @@ After registration, the user will connect automatically.
 
 Registered users wishing to add an existing account to Adium should enter the MongooseIM server’s IP address in the **Connect Server** field on the **Options** tab.
 
+### Pidgin (Ubuntu & CentOS)
+
+1. 	Launch Pidgin.
+2. 	Click the Add button to configure your account.
+3. 	In the **Basic** tab choose **XMPP** as protocol,
+		then enter Username, Domain and Password and click **Add**.
+		After Registration the user will connect automatically.
+
+### Gajim (Ubuntu & CentOS)
+
+The following steps assumes that you have already registered a user on the MongooseIM server, see section **Registering a user** above.
```

Comment:
The following steps assume that you have already registered a user on the MongooseIM server (see section **Registering a user** above).